Source/WebKit/UIProcess/XR/openxr/OpenXRInputSource.cpp

```diff
@@ -143,7 +143,21 @@ XrResult OpenXRInputSource::suggestBindings(SuggestedBindings& bindings) const
         }
     };
 
+    auto isInteractionPathSupported = [](const ASCIILiteral& path) {
+        if (path == handInteractionProfileName) {
+#if defined(XR_EXT_hand_interaction)
+            return OpenXRExtensions::singleton().isExtensionSupported(XR_EXT_HAND_INTERACTION_EXTENSION_NAME ""_span);
+#else
+            return false;
+#endif
+        }
+        return true;
```

Comment:
```suggestion
#if defined(XR_EXT_hand_interaction)
        if (path == handInteractionProfileName) {
            return OpenXRExtensions::singleton().isExtensionSupported(XR_EXT_HAND_INTERACTION_EXTENSION_NAME ""_span);
        }
#else
        UNUSED_PARAM(path);
#endif
        return true;
```

Do you expect to keep populating `isInteractionPathSupported` with more choices? Depending on the answer we might move the ifdef around

Reply:
Yes, the idea would be to have more of this as we support more devices and extensions